addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -305,3 +305,7 @@ $o-wblog-loader-size: 50px;
         background-color: rgba(black, 0.7);
     }
 }
+
+#o_wblog_index_content .container .row .border-end {
+    width:auto
+}
```

Comment:
- I would put `>` between everything to be even more precise.
- Is it not possible to ensure that we have the layout list and the sidebar?
- Also a space and a semicolon are missing in the CSS rule ;)